CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 Changelog
 =========
 
-## [Unreleased]
+### Added
+
+* port_channel property to `Cisco::Interface` class
```

Comment:
Don't add this here, there's already an 'Added' section further down. Also, match the format of the other "Extend" entries:

```
### Added

* `Cisco::UnsupportedError` exception class, raised when a command is explicitly marked as unsupported on a particular class of nodes.
* Extend cisco_vrf with `vni` attribute
+* Extend cisco_interface with `channel_group` property.
```